docs/security.md

```diff
@@ -154,8 +149,15 @@ authorization:
   ldapConfigPath: '<ldap_config_path>'
 ```
 
-The LDAP config file should have the following contents:
+**Caution:**
+The regex pattern matching for roles only works for API calls, and not on the webapp. 
+The webapp only supports ADMIN, USER, and API roles. 
```

Comment:
Webapp is using API, so the security setting should be consistent between webapp and API.
APIs are annotated by `@RolesAllowed("ADMIN/User/API")`. No other roles are supported.